docker-entrypoint.sh

```diff
@@ -59,14 +59,17 @@ if [ ! -d "/opt/frontend/my-volto-project/src/addons/$GIT_NAME" ]; then
   cd /opt/frontend/my-volto-project/
 fi
 
-sed -i "s#\$GIT_NAME#$GIT_NAME#g" jest.config.js
 node /jsconfig $PACKAGE addons/$GIT_NAME/src
 
+if [ ! -d "/opt/frontend/my-volto-project/src/addons/$GIT_NAME/jest-addon.config.js" ]; then
```

Comment:
I don't this condition is correct, right now it tests if the directory `..jest-addon.config.js` doesn't exist.